packages/js/src/novu.ts

```diff
@@ -39,21 +39,21 @@ export class Novu implements Pick<NovuEventEmitter, 'on'> {
     this.#emitter = new NovuEventEmitter();
     this.#session = new Session(
       {
-        applicationIdentifier: options.applicationIdentifier,
-        subscriberHash: options.subscriberHash,
+        applicationIdentifier: 'applicationIdentifier' in options ? options.applicationIdentifier : undefined,
+        subscriberHash: 'subscriberHash' in options ? options.subscriberHash : undefined,
         subscriber: buildSubscriber(options),
       },
       this.#inboxService,
       this.#emitter
     );
     this.#session.initialize();
     this.notifications = new Notifications({
-      useCache: options.useCache ?? true,
+      useCache: 'useCache' in options && options.useCache !== undefined ? options.useCache : true,
```

Comment:
```suggestion
      useCache: options.useCache ?? true,
```

I think this should work too?

Reply:
done